src/prod/linuxsetup/runtime/scripts/servicefabricupdater.service

```diff
@@ -5,6 +5,7 @@ Description=ServiceFabric Updater Daemon
 Type=simple
 EnvironmentFile=/etc/servicefabric/servicefabricupdater.config
 ExecStart=/opt/microsoft/servicefabric/bin/doupgrade.sh $PathToTargetInfoFile
+ExecStartPost=/bin/sh -c 'umask 022; pgrep starthost.sh > /var/run/servicefabricupdater.pid'
```

Comment:
Should this be `doupgrade.sh` instead of `starthost.sh`?

Reply:
It should, I apparently had it as a local change that didn't get committed.

Reply:
+1